benchmarks/benchmark/tools/locust-load-inference/main.tf

```diff
@@ -34,7 +34,7 @@ locals {
     [for data in split("---", templatefile(manifest_file, {
       artifact_registry              = var.artifact_registry
       namespace                      = var.namespace
-      inference_server_service       = var.inference_server_service
+      inference_server_service       = var.inference_server.service
```

Comment:
@Bslabe123 this change will break locust automation - let's revert this file, since the latency profiler is a separate tool